packages/schema/src/core/models/Field.ts

```
@@ -0,0 +1,507 @@
+
+  get required() {
+    const validators = isArr(this.validator) ? this.validator : parseValidatorDescriptions(this.validator);
+    return validators.some((desc) => !!desc?.['required']);
+  }
+
+  get validateStatus() {
+    if (this.validating) return 'validating';
+    if (this.selfInvalid) return 'error';
+    if (this.selfWarnings.length) return 'warning';
+    if (this.selfSuccesses.length) return 'success';
+  }
```

Biome (1.9.4)

[error] This getter should return a value.



(lint/suspicious/useGetterReturn)